web/src/message_events.ts

```diff
@@ -812,6 +813,12 @@ export function update_messages(events: UpdateMessageEvent[]): void {
                 post_edit_topic,
                 post_edit_stream_id,
             );
+        } else {
+            // Since the message content is edited, it could be possible
+            // that the cached asset props of media elements may also be
+            // updated. Hence, we invalidate the older ones to compute
+            // new properties when assets are opened in lightbox again.
+            maybe_invalidate_asset_map_of_message(event.message_id);
```

Comment:
I think we can rename this to just `invalidate_asset_map_of_message`, since the function logic is deterministic. The comment already explains the uncertainty around when this task gets triggered. What do you think?